tests/test_roster_log_review_queue.py

```
@@ -0,0 +1,104 @@
+    count_cf_groups,
+    load_cf_markers,
+)
+from triage.roster_log_review_queue.run import run
+
+
+def _marker_present(xlsx_bytes: bytes, sheet: str) -> bool:
+    markers = load_cf_markers()
+    with zipfile.ZipFile(io.BytesIO(xlsx_bytes), "r") as z:
+        from triage.xlsx_utils import read_text, sheet_name_map
+
+        part = [p for p, n in sheet_name_map(z).items() if n == sheet][0]
```

Ruff (0.15.15)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)